.github/workflows/integration.yml

```diff
@@ -46,40 +46,43 @@ jobs:
         echo "Checking out GeoServer"
         mkdir geoserver
         git clone https://github.com/geoserver/geoserver.git geoserver
-        echo "Checking out GeoFence"
-        mkdir geofence
-        git clone https://github.com/geoserver/geofence.git geofence
-        echo "Checking out mapfish-print-v2"
+# uncomment when geofence is building against geotools snapshot
+#        echo "Checking out GeoFence"
+#        mkdir geofence
+#        git clone https://github.com/geoserver/geofence.git geofence
+#        echo "Checking out mapfish-print-v2"
         mkdir mapfish-print-v2
         git clone https://github.com/mapfish/mapfish-print-v2.git mapfish-print-v2
         echo "Checking out mapfish-print-v3"
-        mkdir mapfish-print-v3
-        git clone -b opengis-migration https://github.com/mapfish/mapfish-print.git mapfish-print-v3
-    - name: Apply org.opengis removal scripts
-      run: |
-        cd ~
-        find . -name "remove-opengis.xml"
-        cd ~/geowebcache/geowebcache
-        cp ~/work/geotools/geotools/docs/user/welcome/files/remove-opengis.xml .
-        ant -noinput -buildfile remove-opengis.xml
-        cd ~/geoserver/src
-        cp ~/work/geotools/geotools/docs/user/welcome/files/remove-opengis.xml .
-        ant -noinput -buildfile remove-opengis.xml
-        cd ~/geofence        
-        cp ~/work/geotools/geotools/docs/user/welcome/files/remove-opengis.xml .
-        ant -noinput -buildfile remove-opengis.xml
-        cd ~/mapfish-print-v2
-        cp ~/work/geotools/geotools/docs/user/welcome/files/remove-opengis.xml .
-        ant -noinput -buildfile remove-opengis.xml
-        cd ~/mapfish-print-v3/core/src
-        cp ~/work/geotools/geotools/docs/user/welcome/files/remove-opengis.xml .
-        ant -noinput -buildfile remove-opengis.xml
-        cd ~/mapfish-print-v3/docs/src
-        cp ~/work/geotools/geotools/docs/user/welcome/files/remove-opengis.xml .
-        ant -noinput -buildfile remove-opengis.xml
-        cd ~/mapfish-print-v3/examples/src
-        cp ~/work/geotools/geotools/docs/user/welcome/files/remove-opengis.xml .
-        ant -noinput -buildfile remove-opengis.xml
+# additional check during remove-opengis activity
```

Comment:
I think this block can probably be removed (not just commented out). We can get it back from git if needed.